### PR TITLE
Add standalone project for generated C++ code (TTNN->EmitC)

### PR DESCRIPTION
**Pull request by svuckovicTT**
- Add standalone project (`tools/ttnn-standalone/`)
- Allows for plug&play execution of generated code in TTNN->EmitC path
- Actual generated code is not yet ready for compilation
- Example code that can run is located in `tools/ttnn-standalone/ttnn-standalone.cpp`

### CODE REVIEW

File: tools/ttnn-standalone/ttnn-standalone.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  auto y = ttnn::full(ys, 6.0f, ttnn::DataType::BFLOAT16, ttnn::Layout::TILE);
+
+  // Run fwd pass on device
+  //
+  ttnn::Tensor result = forward(x, y);
+
+  // Print result
+  //
+  result.print();
+}
```
Comment (svuckovicTT) [inline tools/ttnn-standalone/ttnn-standalone.cpp:69]: Keep in mind that all of this is a WIP. Currently I'm testing with a unit test that doesn't have any tensor values provided, so I made up the inputs, and hardcoded some values in `forward` fn.

Once the generated code is actually compile-able, this setup will be a bit different.

File: tools/ttnn-standalone/pch.hpp
Comment (nsmithtt) [inline tools/ttnn-standalone/pch.hpp:1]: Maybe we should call this `ttnn_standalone.hpp` or `ttnn_api.hpp` or `ttnn_includes.hpp` or something?  Is this even a pch, I didn't see anything in the cmake?

**Review by nsmithtt — APPROVED** [review]
Looks great! One minor nit about naming

File: tools/ttnn-standalone/pch.hpp
Comment (svuckovicTT) [reply tools/ttnn-standalone/pch.hpp:1]: Yeah, I'll rename it!

If I understand correctly, the last line in cmake should handle this, do you think there's more to be done?
`target_precompile_headers(ttnn-standalone PRIVATE pch.hpp)`

**Review by sdjordjevicTT — APPROVED** [review]

File: tools/ttnn-standalone/pch.hpp
Comment (nsmithtt) [reply tools/ttnn-standalone/pch.hpp:1]: Ah I missed that! Ok nvm cool.  Thanks! And the rename lgtm

**Pull request merged**